aspnetcore/blazor/forms-validation.md

```diff
@@ -368,6 +366,31 @@ Assign a custom template to <xref:Microsoft.AspNetCore.Components.Forms.InputDat
 
 ::: moniker-end
 
+## Basic validation
+
+In basic form validation scenarios, an <xref:Microsoft.AspNetCore.Components.Forms.EditForm> instance can use declared <xref:Microsoft.AspNetCore.Components.Forms.EditContext> and <xref:Microsoft.AspNetCore.Components.Forms.ValidationMessageStore> instances to validate form fields. A handler for the <xref:Microsoft.AspNetCore.Components.Forms.EditContext.OnValidationRequested> event of the <xref:Microsoft.AspNetCore.Components.Forms.EditContext> executes custom validation logic. The handler's result updates the <xref:Microsoft.AspNetCore.Components.Forms.ValidationMessageStore> instance.
+
+> [!WARNING]
+> When used the same component, the approach in this section is incompatible with the use of a *validator component*, which is described in the [Validator components](#validator-components) section.
+>
+> Use of both approaches simultaneously leads to a conflict over control of the form's <xref:Microsoft.AspNetCore.Components.Forms.ValidationMessageStore>. Basic form validation is most useful in cases where the form's model is defined within the component hosting the form, either as members directly on the component or in a subclass. Use of a validator component is a better choice where an independent model class is used across several components.
```

Comment:
I'm not sure what this is trying to get at. You can have n message stores for a given EditContext and that's how it's designed. They shouldn't be conflicting with each other (you generally can't get to a store owned by another implementation since it's private to it)